res/css/views/beta/_BetaCard.pcss

```diff
@@ -15,8 +15,8 @@ limitations under the License.
 */
 
 .mx_BetaCard {
-    margin-bottom: 20px;
-    padding: 24px;
+    margin-bottom: $spacing-20;
+    padding: $spacing-24;
```

Comment:
it feels like these changes to `mx_BetaCard` are separate to the removal of `mx_BetaDot` so should be in a separate PR?